gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -87,11 +87,16 @@ hook.Add("Think", "TTT2PlayerSprinting", function()
 		if not ply.sprintTS then continue end
 
 		local timeElapsed = CurTime() - ply.sprintTS
-
-		if not ply.sprintMultiplier then
-			ply.sprintProgress = math.min((ply.oldSprintProgress or 0) + timeElapsed * GetGlobalFloat("ttt2_sprint_stamina_regeneration"), 1)
+		local isSprinting = (ply.sprintMultiplier ~= nil)
```

Comment:
theoretically, there already exists a client var: https://github.com/TTT-2/TTT2/blob/d387992086fd9c4fe4669afaa2670ee9bd6aa23a/gamemodes/terrortown/gamemode/shared/sh_sprint.lua#L57

Reply:
Does this exist on the server, however? doing something like `local isSprinting = CLIENT and LocalPlayer().isSprinting or (ply.sprintMultiplier ~= nil)` seems awkward.

Reply:
https://github.com/TTT-2/TTT2/blob/d387992086fd9c4fe4669afaa2670ee9bd6aa23a/gamemodes/terrortown/gamemode/shared/sh_sprint.lua#L46 it's serverside too, same as ply.sprintMultiplier